### PR TITLE
Try build jobs

### PR DESCRIPTION
**Pull request by agjohnson**
I was mostly curious about what this would look like, but it was rather easy to accomplish.

This enables building with Pelican and uses a dummy Sphinx configuration to mostly skip the Sphinx build step. The one hurdle was knowing you needed to put the files in a certain location to get them cloned to RTD storage.

For this use case, build.jobs + a generic builder that skipped Sphinx and collected my build files would be enough -- so probably no metadata.yaml or build.commands required

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    python: "3.8"
+    nodejs: "14"
+  jobs:
+    pre_build:
+      - pip install -r requirements.txt
+      - npm ci
+      - npm run build-html
+    post_build:
+      - rm -rfv ../../artifacts/${READTHEDOCS_VERSION}/sphinx
+      - mv -v ./output/ ../../artifacts/${READTHEDOCS_VERSION}/sphinx
```
Comment (humitos) [inline .readthedocs.yml:18]: Haha! Nice hack to support building with pelican 😛

Comment (ericholscher) [reply .readthedocs.yml:18]: Yea, I figured there would be a blog post about how to support other tools like this in <3 months after we promote this.

**Comment by agjohnson** [comment]
I put this PR up mostly to show @humitos I was doing evil things with his new creation. I'm excited to just roll with this for now though, I'll see how it goes! :upside_down_face:

**Comment by ericholscher** [comment]
@agjohnson does this mean we can host our landing pages on RTD in prod?

**Comment by agjohnson** [comment]
I thought about that, yeah. I'll give it a whirl and see if there is some set up that makes sense here.

**Pull request merged**